src/FortuneWindow.cpp

```diff
@@ -12,76 +13,62 @@
 #define M_ABOUT_REQUESTED 'abrq'
 
 FortuneWindow::FortuneWindow(void)
-	: BWindow(BRect(0,0,300,300),"Fortune",B_DOCUMENT_WINDOW,B_ASYNCHRONOUS_CONTROLS),
+	:
+	BWindow(BRect(0, 0, 300, 400), "Fortune", B_TITLED_WINDOW,
+		B_AUTO_UPDATE_SIZE_LIMITS | B_NOT_ZOOMABLE | B_ASYNCHRONOUS_CONTROLS),
 	fFortune()
 {
-	BPath path;	
+	BPath path;
 	find_directory(B_SYSTEM_DATA_DIRECTORY, &path);
 	path.Append("fortunes");
 
 	fFortune.SetFolder(path.Path());
 
-	BView *back = new BView(Bounds(),"background",B_FOLLOW_ALL, B_WILL_DRAW);
-	back->SetViewColor(ui_color(B_PANEL_BACKGROUND_COLOR));
-	AddChild(back);
+	BButton* next = new BButton("nextbutton", "Get Another",
+		new BMessage(M_GET_ANOTHER_FORTUNE));
 
-	BButton *close = new BButton(BRect(0,0,1,1),"closebutton","Close",
-								new BMessage(B_QUIT_REQUESTED), B_FOLLOW_RIGHT | B_FOLLOW_BOTTOM);
-	close->ResizeToPreferred();
-	close->MoveTo(Bounds().right - 15 - close->Frame().Width(), 
-				Bounds().bottom - 15 - close->Frame().Height());
-	back->AddChild(close);
-	close->MakeDefault(true);
-	
-	BButton *next = new BButton(BRect(0,0,1,1),"nextbutton","Get Another",
-								new BMessage(M_GET_ANOTHER_FORTUNE), B_FOLLOW_RIGHT | B_FOLLOW_BOTTOM);
-	next->ResizeToPreferred();
-	next->MoveTo(close->Frame().left - 15 - next->Frame().Width(), 
-				Bounds().bottom - 15 - next->Frame().Height());
-	back->AddChild(next);
-	
-	BButton *about = new BButton(BRect(0,0,1,1),"aboutbutton","About" B_UTF8_ELLIPSIS,
-								new BMessage(M_ABOUT_REQUESTED), B_FOLLOW_LEFT | B_FOLLOW_BOTTOM);
-	about->ResizeToPreferred();
-	about->MoveTo(next->Frame().left - 15 - about->Frame().Width(), 
-				Bounds().bottom - 15 - about->Frame().Height());
-	back->AddChild(about);
-	
-	
-	BRect r(15,15,Bounds().right - B_V_SCROLL_BAR_WIDTH - 15, next->Frame().top - 15);
-	fTextView = new BTextView(r,"textview",r.OffsetToCopy(0,0).InsetByCopy(10,10),B_FOLLOW_ALL,
-								B_WILL_DRAW | B_PULSE_NEEDED | B_FRAME_EVENTS);
+	BButton* about = new BButton("aboutbutton", "About" B_UTF8_ELLIPSIS,
+		new BMessage(M_ABOUT_REQUESTED));
+
+	fTextView = new BTextView("textview", B_WILL_DRAW | B_PULSE_NEEDED | B_FRAME_EVENTS);
 	fTextView->MakeEditable(false);
-	
-	BScrollView *sv = new BScrollView("scrollview",fTextView,B_FOLLOW_ALL,0,false,true);
-	back->AddChild(sv);
-	
+	fTextView->SetInsets(10, 10, 10, 10);
+
+	BScrollView* sv = new BScrollView("scrollview", fTextView, 0, false, true);
+
 	BString fortune;
 	status_t status = fFortune.GetFortune(&fortune);
-	if(status == B_OK)
+
+	if (status == B_OK)
 	{
 		BString title;
 		fFortune.LastFilename(&title);
 		title.Prepend("Fortune: ");
```

Comment:
Terribly sorry....  here too "Fortuna".

Reply:
I strongly believe that this is a deliberate design choice. Do I still move on with it?

Reply:
Well, since you're feeling strongly, and I even believe you're right... :)
Let's merge as is.

Reply:
Yeah, I wouldn't really insist on a simple line change for any other reason.

It's that the part after `Fortune:` gets replaced with the topic relevant to the displayed fortune. I doubt that "Fortuna" is meant to be used as a noun generally speaking. It also seemed to be an original design choice.